erdgo/blockchain/proxy.go

```diff
@@ -205,8 +215,7 @@ func (ep *elrondProxy) getHTTP(endpoint string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	client := http.DefaultClient
-	response, err := client.Do(request)
+	response, err := Client.Do(request)
```

Comment:
why this change?

Reply:
Injecting a mock http client allows this method to be tested.